website/docs/d/managed_disk.html.markdown

```diff
@@ -110,5 +110,7 @@ resource "azurerm_virtual_machine" "test" {
 * `source_resource_id` - ID of an existing managed disk that the current resource was created from.
 * `os_type` - The operating system for managed disk. Valid values are `Linux` or `Windows`
 * `disk_size_gb` - The size of the managed disk in gigabytes.
+* `disk_iops_read_write` - The number of IOPS allowed for this disk. One operation can transfer between 4k and 256k bytes.  Only applicable to disks of type `UltraSSD`.
```

Comment:
```suggestion
* `disk_iops_read_write` - The number of IOPS allowed for this disk. One operation can transfer between 4k and 256k bytes.
```

Reply:
@katbyte , may I suggest to keep the "_Only applicable to disks of type UltraSSD_" since it is a matter of fact. Setting this value against other disk types won't take effect so we would like to explicitly have users be aware of this.